internal/render/pod.go

```diff
@@ -205,6 +208,48 @@ func (p *Pod) defaultRow(pwm *PodWithMetrics, row *model1.Row) error {
 	return nil
 }
 
+// Calculates the total number of GPUs requested by a pod
+func calculatePodGPUs(spec *v1.PodSpec) string {
```

Comment:
We should have tests on this to make sure we are output the expected results under various conditions.

Reply:
didn't work for me